.azure/scripts/osx-python.sh

```diff
@@ -44,4 +44,3 @@ sudo ln -s /usr/local/bin/python$PYTHON_VERSION /usr/local/bin/python
 which python
 python --version
 python -m ensurepip
-python -m pip install setuptools wheel
```

Comment:
This isn't needed since it is only used in the release.yml, and that uses `pip`, which install the necessary build deps.